services/ui-src/src/components/fields/ChoiceField.tsx

```diff
@@ -1,51 +1,86 @@
 import { useFormContext } from "react-hook-form";
 // components
 import { Choice as CmsdsChoice } from "@cmsgov/design-system";
-import { Box } from "@chakra-ui/react";
+import { Box, Text } from "@chakra-ui/react";
 // utils
 import { makeMediaQueryClasses } from "utils";
 import { InputChangeEvent, AnyObject } from "types";
+import { useEffect, useState } from "react";
 
-export const ChoiceField = ({
-  name,
-  type,
-  value,
-  label,
-  sxOverride,
-  ...props
-}: Props) => {
+export const ChoiceField = ({ name, label, sxOverride, ...props }: Props) => {
   const mqClasses = makeMediaQueryClasses();
+  const singleBoxClass = "single-box";
+
+  const [displayValue, setDisplayValue] = useState<boolean>(false);
 
   // get the form context
   const form = useFormContext();
+  form.register(name);
+
+  // set initial display value to form state field value or hydration value
+  const hydrationValue = props?.hydrate;
+  useEffect(() => {
+    // if form state has value for field, set as display value
+    const fieldValue = form.getValues(name);
+    if (fieldValue) {
+      setDisplayValue(fieldValue);
+    }
+    // else if hydration value exists, set as display value
+    else if (hydrationValue) {
+      setDisplayValue(hydrationValue);
+      form.setValue(name, hydrationValue, { shouldValidate: true });
+    }
+  }, [hydrationValue]); // only runs on hydrationValue fetch/update
```

Comment:
Suggestion: Because of the nature of this field and the fact that the field value is not really used, I actually think we can simplify this a lot. Let me know when you're free and we can pair on it! 👍